synapseclient/models/curation.py

```diff
@@ -226,6 +258,74 @@ def store(self, *, synapse_client: Optional[Synapse] = None) -> "CurationTask":
         Returns:
             CurationTask: The CurationTask object.
 
+        Example: Create a new file-based curation task
+            &nbsp;
+
+            ```python
+            from synapseclient import Synapse
+            from synapseclient.models import CurationTask, FileBasedMetadataTaskProperties
+
+            syn = Synapse()
+            syn.login()
+
+            # Create file-based task properties
+            file_properties = FileBasedMetadataTaskProperties(
+                upload_folder_id="syn1234567",
+                file_view_id="syn2345678"
+            )
+
+            # Create the curation task
+            task = CurationTask(
+                project_id="syn9876543",
```

Comment:
Nit: this should probably be changed to a test folder id?

Reply:
This is just an example Synapse ID. `9876543` is just a set of descending numbers.

> I noticed you can start a grid session with grid.create(). But how do you edit the grid programmatically in Python? Or the grid will have to be edited in the UI? And is it possible to have multiple active grid sessions? If so, which session will you get when you run grid.create(attach_to_previous_session=True)?

There aren't many things that you can currently do with the grid that we needed to expose in the Python client, so I didn't include these items.

You can have multiple active grid sessions.

When you try to attach to a previous session and it was started with a record set, it will attach back to the first found that it matches.

Reply:
Got it Bryan. Thanks for explaining.